configure.ac

```diff
@@ -981,6 +982,42 @@ if test -z "$LEX" || \
     fi
 fi
 
+# If we need to generate man pages, we need pandoc >v1.19.
+AC_PATH_PROG([PANDOC], [pandoc])
+
+# If we found Pandoc, check its version.  We need >=v1.19.
+# To be clear: I know that v1.19 works, and I know that v1.9 does not
+# work.  I did not test the versions in between to know exactly what
+# the lowest version is that works.  Someone is free to update this
+# check someday to be more accurate if they wish.
+min_major_version=1
+min_minor_version=19
+AS_IF([test -n "$PANDOC"],
```

Comment:
It's our intention that users do not need pandoc to build OMPI, but with this configure check they will need to have it installed (right?) to configure/make/make-install OMPI from a tarball. Is there a way to make this check conditional if the man pages were created?

Reply:
FWIW, we do the same thing for flex (i.e., we always check for it, but we only error if a file that will be generated by flex is not already present -- i.e., if we're in a distribution tarball that already contains the generated file, then we don't need flex).

Reply:
Ok. I wasn't sure how that was handled. It read like the user would need pandoc even when using a distribution tarball.